seatunnel-connectors-v2/connector-redis/src/main/java/org/apache/seatunnel/connectors/seatunnel/redis/config/RedisConfig.java

```diff
@@ -36,6 +36,12 @@ public enum HashKeyParseMode {
         KV;
     }
 
+    public static final Option<RedisVersion> REDIS_VERSION =
+            Options.key("redis_version")
+                    .enumType(RedisVersion.class)
+                    .noDefaultValue()
```

Comment:
Please set default version to version 7.

Reply:
In the next commit, this configuration is removed because we get the version through the client.